react/features/invite/functions.js

```diff
@@ -275,15 +275,20 @@ export function invitePeopleAndChatRooms( // eslint-disable-line max-params
     }
 
     return new Promise((resolve, reject) => {
-        $.post(
-                `${inviteServiceUrl}?token=${jwt}`,
-                JSON.stringify({
-                    'invited': inviteItems,
-                    'url': inviteUrl
-                }),
-                resolve,
-                'json')
-            .fail((jqxhr, textStatus, error) => reject(error));
+        $.ajax({
```

Comment:
If you're changing stuff and testing with this much change, maybe you should just make it use fetch?

Reply:
done!